components/index/ValueProps.tsx

```diff
@@ -0,0 +1,49 @@
+import { Grid, GridItem } from '@patternfly/react-core';
+
+import valuePropsList from '@components/index/valuePropsList';
+
+const ValueProps = (
+  <Grid hasGutter>
+    {valuePropsList.map((valueProp, index) => (
+      <>
+        <GridItem span={1}></GridItem>
+        {index % 2 === 0 ? (
+          <></>
+        ) : (
+          <GridItem span={5} style={{ padding: '2rem' }}>
+            picture
+          </GridItem>
+        )}
+        <GridItem span={5} style={{ padding: '2rem' }}>
+          <h2
+            style={{
+              fontSize: '1.3rem',
+              color: 'var(--pf-global--link--Color)',
+            }}
+          >
+            <b>{valueProp[0]}</b>
+          </h2>
+          <br />
+          <div
+            style={{
+              color: 'var(--pf-global--Color--200)',
+              paddingRight: '1rem',
+            }}
+          >
+            {valueProp[1]}
+          </div>
+        </GridItem>
+        {index % 2 === 0 ? (
+          <GridItem span={5} style={{ padding: '2rem' }}>
+            picture
```

Comment:
It may take a while before we can get some graphics created. For the time being, we can just put each value prop as two columns side by side, i.e. remove the picture holder. You can save this placeholder version somewhere so that we can use this layout after we have the pictures available.